plio/cache.py

```diff
@@ -0,0 +1,34 @@
+from django.core.cache import cache
+
+
+def get_cache_key(instance):
+    """Calculates cache key for an instance"""
+    instance_class = instance.__class__.__name__
+    return {
+        "Video": f"video_{instance.pk}",
+        "Plio": f"plio_{instance.pk}",
+        "Session": f"session_{instance.pk}",
+        "User": f"user_{instance.pk}",
+        "UserMeta": f"user_meta_{instance.pk}",
+    }.get(instance_class, None)
+
+
+def invalidate_cache_for_instance(instance):
+    """Deletes cache for a particular instance"""
+    cache_key = get_cache_key(instance)
+    if cache_key:
+        cache.delete(cache_key)
```

Comment:
won't this throw an error if the key is not in the cache?

Reply:
It won't. It simply returns True or False.